Alerting: Update documentation for MS Teams integration (#90610)
grafana/grafana: docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-teams.md

## Before you begin

> To set up Microsoft Teams for integration with Grafana Alerting, you need to create a new Workflow that accepts a webhook requests. This allows Grafana to send alert notifications to Microsoft Teams channels.

Reviewer: ```suggestion
To set up Microsoft Teams for integration with Grafana Alerting, create a new workflow that accepts Webhook requests. This allows Grafana to send alert notifications to Microsoft Teams channels.
```

> ### Create a new Workflow in Microsoft Teams

Reviewer: ```suggestion
### Create a workflow in Microsoft Teams
```

> 1. To create a new workflow, follow the steps in [Create flows in Microsoft Teams](https://learn.microsoft.com/en-us/power-automate/teams/teams-app-create) documentation. 

Reviewer: hey - we need to use repetitive list numbering - see here: https://grafana.com/docs/writers-toolkit/write/markdown-guide/#numbered-lists - so just 1. 1. 1. will render 1.2.3.

> 1. To create a new workflow follow the steps in [Create flows in Microsoft Teams](https://learn.microsoft.com/en-us/power-automate/teams/teams-app-create)
> 2. Microsoft provides template library. You can use template "Post to a channel when a webhook request is received" 
> 3. At the end of workflow creation wizard, copy the provided URL

Reviewer 2: ```suggestion
1. To create a new workflow, follow the steps in [Create flows in Microsoft Teams](https://learn.microsoft.com/en-us/power-automate/teams/teams-app-create) documentation. 
2. Consider using the template "Post to a channel when a webhook request is received" from the Microsoft template library. 
3. After creating the workflow in Teams, copy the provided URL for use in the next steps.
```

> 1. To create a new workflow follow the steps in [Create flows in Microsoft Teams](https://learn.microsoft.com/en-us/power-automate/teams/teams-app-create)
> 2. Microsoft provides template library. You can use template "Post to a channel when a webhook request is received" 
> 3. At the end of workflow creation wizard, copy the provided URL

Reviewer: ```suggestion
1. To create a new workflow follow the steps in [Create flows in Microsoft Teams](https://learn.microsoft.com/en-us/power-automate/teams/teams-app-create)
1. Microsoft provides template library. You can use template "Post to a channel when a webhook request is received" 
1. At the end of workflow creation wizard, copy the provided URL
```

> 1. To create a new workflow follow the steps in [Create flows in Microsoft Teams](https://learn.microsoft.com/en-us/power-automate/teams/teams-app-create)
> 2. Microsoft provides template library. You can use template "Post to a channel when a webhook request is received" 
> 3. At the end of workflow creation wizard, copy the provided URL

Reviewer: ```suggestion
1. To create a new workflow, follow the steps in [Create flows in Microsoft Teams](https://learn.microsoft.com/en-us/power-automate/teams/teams-app-create).
2. Microsoft provides a template library. You can use the template **Post to a channel when a webhook request is received**.
3. At the end of workflow creation wizard, copy the URL that is provided.
```
